django_loci/base/admin.py

```diff
@@ -278,7 +281,7 @@ def _get_floorplan_instance(self):
         floorplan = data.get('floorplan') or self.floorplan_model()
         floorplan.location = instance.location
         floor = data.get('floor')
-        floorplan.floor = floor if floor is not None else floorplan.floor
+        floorplan.floor = floor
```

Comment:
are you sure this should be changed?

Reply:
As @atb00ker said above,

> However, I noticed if floor number is set and then I edit the form to remove the floor number and save, the change is not honored and the hold floor number is retained.

So I had to make this change in order to fix that.